cmd/protoc-gen-go-tableau-loader/helper/helper.go

```diff
@@ -46,22 +45,33 @@ func ParseGoType(file *protogen.File, fd protoreflect.FieldDescriptor) string {
 	case protoreflect.BytesKind:
 		return "[]byte"
 	case protoreflect.MessageKind:
-		fullName := string(fd.Message().FullName())
-		pkg := string(file.Desc.Package())
-		protoName := fullName
-		if strings.HasPrefix(fullName, pkg) {
-			// defined at the same package
-			protoName = fullName[len(pkg)+1:]
+		if file, ok := gen.FilesByPath[fd.Message().ParentFile().Path()]; ok {
+			message := FindMessageByDescriptor(file.Messages, fd.Message())
+			if message != nil {
+				return message.GoIdent.GoName
+			}
 		}
-		goName := strings.ReplaceAll(protoName, ".", "_")
-		return string(file.GoImportPath.Ident(goName).GoName)
+		return fmt.Sprintf("<not found:%s>", fd.Message().FullName())
```

Comment:
- encapsulate a function: `FindMessage(md protoreflect.MessageDescriptor) *protogen.Message`
- panic: `panic(fmt.Sprintf("not found: %s", fd.Message().FullName()))`